### ISSUE TITLE
Enhancement: Opportunity to do additional parallel data fetches at login

### ISSUE THREAD

**Issue by sjd78** [oVirt/ovirt-web-ui#1210]
The login code needs to fetch a lot of information to populate the redux store before the app can be allowed to run normally.  A lot of the process is already done with parallel fetches.  There is a small gain to be made by:

  1. Fetch **user permissions** and **server config values** in parallel
  2. Fetch all **server config values** in parallel

This should max out the number of fetches that can be done at the same time (a browser limit).
